src/ParameterSchedulers.jl

```diff
@@ -61,6 +61,15 @@ mutable struct Scheduler{T, O, F} <: Flux.Optimise.AbstractOptimiser
     optim::O
     update_func::F
 end
+function Scheduler(state::IdDict{Any, Int},
+                   schedule::T,
+                   optim::O,
+                   update_func::F) where {T, O, F}
+    @warn """Scheduler will transition to explicit Optimisers.jl style
```

Comment:
Is the `@warn` because deprecations aren't reported by default?

Reply:
There's nothing to redirect the deprecation warning to, right?Otherwise we could put this part and #55 together for a patch release followed by a breaking release.

Reply:
`Base.depwarn` works even without a binding to redirect to. We use it quite a bit already so it should be safe: https://github.com/search?q=org%3AFluxML%20depwarn&type=code